src/yang/ietf-qos-action.yang

```diff
@@ -540,6 +543,24 @@ module ietf-qos-action {
       "Random Early Detect (RED) configuration parameters.";
   }
 
+  grouping codel-config-parameters {
+    leaf target {
+      type uint64;
+      default 5000;
+      description
```

Comment:
Please add a unit statement for microseconds.

Also in the description statement, please s/target/Target.